README.md

```diff
@@ -60,6 +60,14 @@ To run the tests with Clang sanitizers, run the following command from the root
 	./autogen.sh --enable-asan
 	make check
 
+To run the unit tests with leak sanitizers, run the following command from the root of the repository:
+
+	./autogen.sh --enable-asan
+	export ASAN_OPTIONS=detect_leaks=1
+	export ASAN_OPTIONS=fast_unwind_on_malloc=0
```

Comment:
@hughbe do we need this when we compile with ` -fno-omit-frame-pointer`? according to the FAQ (https://github.com/google/sanitizers/wiki/AddressSanitizer#faq) one of them should be enough

Reply:
On my linux VM things only worked when both were set

Reply:
How does the second `export` not override the value from the first one?

Reply:
it doesn't seem to do that on my computer!

Reply:
Ok well it doesn't hurt since it's just in the docs :)